Specs/BSPropertySpec.mm

```diff
@@ -33,10 +41,24 @@
             });
         });
 
-        context(@"when the return type is a non-existent objective-c class", ^{
+        context(@"when the return type is an objective-c class conforming to a protocol", ^{
+            it(@"determines the protocol", ^{
+                property = [BSProperty propertyWithClass:[ClassWithProtocolInstance class] propertyNameString:@"protocolInstance"];
+                expect(property.returnType == [TestProtocolImpl class]).to(equal(YES));
+            });
+        });
+        
+        context(@"when the return type is an objective-c protocol", ^{
+            it(@"determines the protocol", ^{
+                property = [BSProperty propertyWithClass:[ClassWithProtocolProperty class] propertyNameString:@"protocolObject"];
+                expect(property.returnType == @protocol(TestProtocol)).to(equal(YES));
+            });
+        });
+        
+        context(@"when the return type conforms to multiple objective-c protocols", ^{
```

Comment:
Because I'm not sure if there's a way to bind multiple protocols to a class?

There's no established method to specify a key for binding multiple protocols to a class.  i.e. binding protocols to class looks like ``[binder bind:@protocol(TestProtocol) toClass:[Test class]]``

It is however possible for someone to assume that a property conforming to multiple protocols will be automatically bound.  The idea here was to provide contextual feedback as opposed to waiting for a exception on get instance for key.